### PR TITLE
Fix issues with stats sometimes being reversed

### PR DESCRIPTION
**Pull request by alicetouhou**

### CODE REVIEW

File: tests/db/database.test.ts [hunk 1]
```diff
@@ -190,6 +190,30 @@ describe('Test Database', () => {
...
+		await database.insertGame(
+			winnerDeckCode.body,
+			loserDeckCode.body,
+			winner.body.uuid,
+			loser.body.uuid,
+			'player_won',
+			35000000,
+			winner.body.uuid,
+			'123456789ABCDEF',
+			Buffer.from([0x00, 0x0a, 0x00, 0x04, 0x00]),
```
[the remark this answers is not in the document]
Comment (alicetouhou) [reply tests/db/database.test.ts:202]: Actually, nothing, it's going to store replays but as those don't exist yet it doesn't matter what the data is

[the remark this answers is not in the document]
Comment (Lunarmagpie) [reply tests/db/database.test.ts:202]: You should say that then

[the remark this answers is not in the document]
Comment (alicetouhou) [reply tests/db/database.test.ts:202]: "The worst she can say is no"
The pull request review:

[the remark this answers is not in the document]
Comment (Lunarmagpie) [reply tests/db/database.test.ts:202]: bro im barely shedding rn

**Review by Lunarmagpie — APPROVED** [review]
LGTM

**Pull request merged**